mne/epochs.py

```diff
@@ -1066,15 +1066,47 @@ def _compute_aggregate(self, picks, mode='mean'):
     @property
     def _name(self):
         """Give a nice string representation based on event ids."""
+        return self._get_name()
+
+    def _get_name(self, count='frac', ms='×', sep='+'):
+        """Generate human-readable name for epochs and evokeds from event_id.
+
+        Parameters
+        ----------
+        count : 'frac' | 'total'
+            Whether to include the fraction or total number of epochs that each
+            event type contributes to the number of all epochs.
+            Ignored if only one event type is present.
+        ms : str | None
+            The multiplication sign to use. Pass ``None`` to omit the sign.
+            Ignored if only one event type is present.
+        sep : str
+            How to separate the different events names. Ignored if only one
+            event type is present.
+        """
+        _check_option('count', value=count, allowed_values=['frac', 'total'])
+
         if len(self.event_id) == 1:
             comment = next(iter(self.event_id.keys()))
         else:
             count = Counter(self.events[:, 2])
```

Comment:
`count` is a param of the function (frac/total) and is overwritten here as the `Counter` result.  Your conditional `if count == 'frac'` below will not work.

Reply:
Good catch! I pushed a fix